cdap-data-fabric/src/main/java/co/cask/cdap/spi/data/sql/PostgresSqlStructuredTableAdmin.java

```diff
@@ -56,17 +59,22 @@ public PostgresSqlStructuredTableAdmin(StructuredTableRegistry registry, DataSou
   @Override
   public void create(StructuredTableSpecification spec) throws IOException, TableAlreadyExistsException {
```

Comment:
Table drop also needs to be made idempotent. It does not have to be done for this release though. Can you file a JIRA and add a TODO to the drop method?

Reply:
👍

Reply:
Created: https://issues.cask.co/browse/CDAP-15068